src/index.js

```diff
@@ -60,7 +60,8 @@ export default class extends PureComponent {
     imgSrc: PropTypes.string,
     saveData: PropTypes.string,
     immediateLoading: PropTypes.bool,
-    hideInterface: PropTypes.bool
+    hideInterface: PropTypes.bool,
+    refreshBackgroundImage: PropTypes.bool,
```

Comment:
Based on another comment, we would then loose the `refreshBackgroundImage` prop, and update every time the imgSrc changes.

```suggestion
    hideInterface: PropTypes.bool
```